src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,14 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+
+        Lottery lottery = new Lottery();
+        Ball ball1 = lottery.getRandomBall();
+        Ball ball2 = lottery.getRandomBall();
+        Ball ball3 = lottery.getRandomBall();
+        System.out.println("Random ball1: " + ball1);
+        System.out.println("Random ball2: " + ball2);
+        System.out.println("Random ball3: " + ball3);
```

Comment:
what would you do if we asked you to create 50000 balls? consider using loop instead